extensions/git/src/model.ts

```diff
@@ -241,12 +241,21 @@ export class Model {
 			}
 
 			const config = workspace.getConfiguration('git');
-			const ignoredRepos = new Set(config.get<Array<string>>('ignoredRepositories'));
 
-			if (ignoredRepos.has(rawRoot)) {
+			const ignoredRepos = (config.get<Array<string>>('ignoredRepositories') || []).map(x => Uri.file(x).fsPath);
```

Comment:
Why converting the path, which can be absolute or relative, into a URI and then getting its fsPath?

Reply:
That change is related to https://github.com/Microsoft/vscode/issues/33498
Without that change, if user type 'C:/asd', it won't ignore repo c:/asd in case insensitive file systems.